sources/tools/Stride.Importer.3D/MeshConverter.cs

```diff
@@ -846,13 +845,16 @@ private unsafe MeshInfo ProcessMesh(Scene* scene, Silk.NET.Assimp.Mesh* mesh, Di
             }
 
             // Build the indices data buffer
-            var nbIndices = 3 * mesh->MNumFaces;
-            byte[] indexBuffer;
+            var nbIndices = (int)(3 * mesh->MNumFaces);
+            byte[] indexBuffer = null;
```

Comment:
Can you rollback this change ? Or move the declaration at line 857 instead

Reply:
decided to move it.